neo.UnitTests/UT_ProtocolSettings.cs

```diff
@@ -0,0 +1,65 @@
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Diagnostics;
+using Neo.Ledger;
+//using Settings = Neo.Plugins.Settings;
+using ProtocolSettings = Neo.ProtocolSettings;
+
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_ProtocolSettings
+    {
+        // standard time: 2 seconds
+        private uint BlockTime = 2;
```

Comment:
I don't quite understand what it does.

Reply:
@erikzhang which part exactly?
UT_ProtocolSettings is a test class that will validate if information is being correctly read from `protocol.json`.
In this case, I set `"SecondsPerBlock": 2` on json, and validate that value matches 2 here. This is important because if we miss this test and protocol is not properly loaded (if it's in different folder, etc), the standard time will appear here (= 15 seconds).
The purpose of reducing time is due to the next tests we will be doing on UT_Consensus, that will require many timeouts, and we don't want it to take long (Akka TestKit only allows 3 seconds wait anyway, so 2 is a good initial move).

Reply:
Then if we modify the configuration file, we have to modify the code here? Doesn't this lose the meaning of the configuration file?

Reply:
The way configuration is now, it is strongly attached to the whole system, in a way that once `protocol.json` is loaded on UnitTest project, there's no way to unload it and try different options.
I think this is not bad in fact, because indeed most of the code depend on its values.
If you prefer, I can remove the 2 seconds test, and it becomes implicit. I was useful to me, because it took time for me to properly load the settings on the project, but now that it's fine and I'm sure of it, we can remove it.